docs/ru-RU/golos.vesting_contract.md

```diff
@@ -298,19 +296,19 @@ void vesting::close(
 Транзакция должна быть подписана аккаунтом `owner`.  
 Для выполнения операции-действия требуется, чтобы баланс всего вестинга, в том числе делегированного, был нулевым.
 
-
-## Операция-действие procwaiting
-Операция-действие `procwaiting` используется для ускорения вывода средств, предназначенных для вывода, и отмены делегирований, предназначенных для отмены. Операция-действие имеет вид:
+## Операция-действие paydelegator
+Операция-действие `paydelegator` является системной, вызывается смарт-контрактом публикации (posting). Смарт-контракт постинг не может вносить изменения в таблицы смарт-контракта вестинг. Поэтому при закрытии поста смарт-контракт постинг вызывает `paydelegator` и через него проводит выплаты делегаторам. Операция-действие `paydelegator` имеет вид:
 ```cpp
-void vesting::procwaiting(
-    symbol symbol,
-    name payer
+void vesting::paydelegator(
```

Comment:
Action `vesting::paydelegator` missing now. Information about `vesting::procwaiting` is worth leaving.